fineract-provider/src/main/resources/sql/migrations/core_db/V343__m_trial_balance_table.sql

```diff
@@ -0,0 +1,35 @@
+--
```

Comment:
V_343 file is added to the code already. So kindly use the next available migration script number